src/UserService.Business/Commands/Certificate/EditCertificateCommand.cs

```diff
@@ -23,111 +25,113 @@
+      IRequestClient<ICreateImagesRequest> rcImage,
+      ILogger<EditCertificateCommand> logger)
+    {
+      _accessValidator = accessValidator;
+      _httpContextAccessor = httpContextAccessor;
+      _userRepository = userRepository;
+      _certificateRepository = certificateRepository;
+      _mapper = mapper;
+      _createImageDataMapper = createImageDataMapper;
+      _rcImage = rcImage;
+      _logger = logger;
+    }
 
-            DbUserCertificate certificate = _certificateRepository.Get(certificateId);
+    public OperationResultResponse<bool> Execute(Guid certificateId, JsonPatchDocument<EditCertificateRequest> request)
+    {
+      List<string> errors = new List<string>();
 
-            if (!(sender.IsAdmin ||
-                  _accessValidator.HasRights(Rights.AddEditRemoveUsers))
-                  && senderId != certificate.UserId)
-            {
-                throw new ForbiddenException("Not enough rights.");
-            }
+      Guid senderId = _httpContextAccessor.HttpContext.GetUserId();
+      DbUser sender = _userRepository.Get(senderId);
 
-            var imageOperation = request.Operations.FirstOrDefault(o => o.path.EndsWith(nameof(EditCertificateRequest.Image), StringComparison.OrdinalIgnoreCase));
-            Guid? imageId = null;
-            if (imageOperation != null)
-            {
-                imageId = GetImageId(JsonConvert.DeserializeObject<AddImageRequest>(imageOperation.value?.ToString()), errors);
-            }
+      DbUserCertificate certificate = _certificateRepository.Get(certificateId);
 
-            JsonPatchDocument<DbUserCertificate> dbRequest = _mapper.Map(request, imageId);
+      if (!(_accessValidator.HasRights(Rights.AddEditRemoveUsers))
+        && senderId != certificate.UserId)
+      {
+        throw new ForbiddenException("Not enough rights.");
+      }
 
-            bool result = _certificateRepository.Edit(certificate, dbRequest);
+      Operation<EditCertificateRequest> imageOperation = request.Operations
+        .FirstOrDefault(o => o.path.EndsWith(nameof(EditCertificateRequest.Image), StringComparison.OrdinalIgnoreCase));
 
-            return new OperationResultResponse<bool>
-            {
-                Status = errors.Any() ? OperationResultStatusType.PartialSuccess : OperationResultStatusType.FullSuccess,
-                Body = result,
-                Errors = errors
-            };
-        }
+      Guid? imageId = null;
+
+      if (imageOperation != null)
+      {
+        imageId = GetImageId(JsonConvert.DeserializeObject<AddImageRequest>(imageOperation.value?.ToString()), errors);
+      }
+
+      JsonPatchDocument<DbUserCertificate> dbRequest = _mapper.Map(request, imageId);
+
+      bool result = _certificateRepository.Edit(certificate, dbRequest);
+
+      return new OperationResultResponse<bool>
+      {
+        Status = errors.Any() ? OperationResultStatusType.PartialSuccess : OperationResultStatusType.FullSuccess,
+        Body = result,
```

Comment:
body = _certificateRepository.Edit(certificate, dbRequest); + remove result variable + body above status

Reply:
will be done in https://github.com/LT-Students/DigitalOffice-UserService/pull/122